community/concepts/grain.md

```diff
@@ -0,0 +1,59 @@
+### Grain
```

Comment:
I should probably take responsibility for this concept -- the posts that this document are based on are fairly out of date, might be cleaner if I write it from scratch.